src/library_html5.js

```diff
@@ -152,6 +152,13 @@ var LibraryHTML5 = {
     },
     
     registerOrRemoveHandler: function(eventHandler) {
+      if (!eventHandler.target) {
+#if ASSERTIONS
+        console.error('registerOrRemoveHandler: the target element for event handler registration does not exist, when processing the following event handler registration:');
```

Comment:
N.b. calling `console.error` here is intentional, because by calling `err`, the result is an unhelpful

![image](https://user-images.githubusercontent.com/225351/225601687-8909e12e-4947-4bc2-a110-4d3cb56e0d3b.png)

but by calling `console.error()`, one gets an informative result

![image](https://user-images.githubusercontent.com/225351/225601756-6f7e8784-5c87-4553-8acb-0a39443a62b3.png)

Reply:
Is this specific to this logging location? Or is it something generic we can fix?

Reply:
I suggest we use `err` here and then fix `err` to do the right thing everywhere.

Reply:
IIRC the plan after landing https://github.com/emscripten-core/emscripten/pull/18017 was to have `err()` go to `console.error` once again.   I guess we (I) didn't get around that?

Reply:
Changed to use err().